windows_containers/windows-containers-release-notes-2-x.adoc

```diff
@@ -17,23 +17,46 @@ The release notes for Red Hat OpenShift for Windows Containers tracks the develo
+This release of the WMCO provides bug fixes and enhancements for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 2.0.0 were released in link:https://access.redhat.com/errata/RHBA-2021:0440[RHBA-2021:0440].
 
 WMCO supports self-managed clusters built using installer-provisioned infrastructure running on the following cloud providers:
 
 * Amazon Web Services (AWS)
 * Microsoft Azure
+* VMware vSphere
 
-The following Windows Server operating systems are supported in the initial release of the WMCO:
+The following Windows Server operating systems are supported in this release of the WMCO:
 
 * Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
 
-[id="wmco-1-0-1-known-issues"]
+The WMCO 2.x version is only compatible with {product-title} 4.7.
+
+[id="wmco-2-0-0-enhancements"]
+=== New features and enhancements
+
+This release adds the following new features and improvements.
+
+[id="wmco-2-0-0-vsphere-support"]
+==== Support for clusters running on VMware vSphere
+
+You can now run Windows nodes on a cluster installed on VMware vSphere version 6.5, 6.7, or 7.0. You can create Windows compute nodes by creating a Windows `MachineSet` object on vSphere to host Windows Server compute machines. For more information, see xref:../windows_containers/creating_windows_machinesets/creating-windows-machineset-vsphere.adoc#creating-windows-machineset-vsphere[Creating a Windows `MachineSet` object on vSphere].
+
+[id="wmco-2-0-0-monitoring"]
+==== Enhanced Windows node monitoring
+
+Windows nodes are now fully integrated with most of the monitoring capabilities provided by the web console. The only gap in monitoring Windows nodes compared to Linux nodes is viewing workload graphs for pods running on Windows nodes.
+
+[id="wmco-2-0-0-known-issues"]
 === Known issues
 
+* Currently, the Filesystem graphs available in the web console do not display for Windows nodes. This is caused by changes in the Filesystem queries. This will be fixed in a future release of WMCO. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1930347[*BZ#1930347*])
+
+* The Prometheus windows_exporter collects metrics through HTTP, so it is considered unsafe. The windows_exporter recently added support for HTTPS configuration. Support for HTTPS configuration in the WMCO will be added in a future release.
+
+// Need to address this. Image version 10.0.17763.1457 is no longer available for AWS.
 * If you have a cluster with two Windows nodes, and you create a web server deployment with two replicas, the pods each land on a Windows compute node. In this scenario, if you create a `Service` object with type `LoadBalancer`, communication with the load balancer endpoint is flaky. To mitigate this issue, you must use Windows Server 2019 with a version 10.0.17763.1457 or earlier. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1905950[*BZ#1905950*])
```

Comment:
I realise that this line is not being changed in this PR, but you could replace the word "flaky" with an explicit description of the symptom.